### PR TITLE
Add custom Layout

### PR DESCRIPTION
**Pull request by adguernier**
Add custom Layout with a custom AppBar

image

### CODE REVIEW

File: src/utils/isEnvModeDev.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const isEnvModeDev = import.meta.env.MODE === "development";
```
Comment (fzaninotto) [inline src/utils/isEnvModeDev.ts]: A one liner isn't worth its own script IMHO

File: src/ui/AppLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Layout, LayoutProps } from "react-admin";
+import { ReactQueryDevtools } from "react-query/devtools";
+import { AppBar } from "./AppBar";
+import { isEnvModeDev } from "../utils";
+
+export const AppLayout = (props: LayoutProps) => {
+  return (
+    <>
+      <Layout {...props} appBar={AppBar} />
+      {isEnvModeDev ? <ReactQueryDevtools /> : null}
```
Comment (fzaninotto) [inline src/ui/AppLayout.tsx]: ReactDevTools are already hidden in production. https://tanstack.com/query/v3/docs/react/devtools#import-the-devtools

File: src/ui/AppBar.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { TitlePortal, AppBar as RaAppBar } from "react-admin";
+import { Box } from "@mui/material";
+export const AppBar = () => {
+  return (
+    <RaAppBar color="transparent">
+      <TitlePortal />
+      <h4>RA E-commerce</h4>
```
Comment (fzaninotto) [inline src/ui/AppBar.tsx]: That's weird: you'll get this *in addition* to the page title. Is it what you want?

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: src/ui/AppBar.tsx (hunk 3 above)
Comment (adguernier) [reply src/ui/AppBar.tsx]: It was to replace the logo. But I can add a logo like in the demo

**Review by djhi — APPROVED** [review]

**Pull request merged**